src/DvfHelpers.php

```diff
@@ -48,7 +49,7 @@ public function transformMachineName($string) {
    */
   public function getHelpPageLink($title) {
```

Comment:
Is the title generally the same? eg "More help" if so maybe `getHelpPageLink($title = 'More help')`

Reply:
Thanks @jez500 I think this is mainly an issue with poor variable naming. The title indicates which help page to load. E.g. "label-overrides". I've update the `$variable_name` and `// comment` accordingly.